cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -398,6 +398,22 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
         val callee = call.callee
         val language = call.language
 
+        // Check for a possible overloaded operator->
+        if (
+            callee.language is HasOperatorOverloading &&
+                callee is MemberExpression &&
+                callee.operatorCode == "->" &&
+                callee.base.type !is PointerType
```

Comment:
We pretty much have the same code for member access. Is it possible to extract it unto a common function?

Reply:
yes, good point. Fixed